mod/isic/spec/set/self/isic_division_formula_spec.rb

```diff
@@ -0,0 +1,12 @@
+# -*- encoding : utf-8 -*-
+
+RSpec.describe Card::Set::Self::IsicDivisionFormula do
+  # described_class not in scope :(
+  let(:dummy_class) { Class.new { include Card::Set::Self::IsicDivisionFormula } }
```

Comment:
This unit test is more "unit-ish" than usual, because this dummy class has nothing to do with cards.  Might be a pattern worth supporting more deeply.

Reply:
good point.